apis/kueue/v1alpha1/workload_webhook_test.go

```diff
@@ -71,11 +72,22 @@ func TestValidateWorkload(t *testing.T) {
 			},
 		},
 		"should have valid priorityClassName": {
-			workload: testingutil.MakeWorkload(testWorkloadName, testWorkloadNamespace).PriorityClass("invalid_class").Obj(),
+			workload: testingutil.MakeWorkload(testWorkloadName, testWorkloadNamespace).
+				PriorityClass("invalid_class").
+				Priority(pointer.Int32(0)).
+				Obj(),
 			wantErr: field.ErrorList{
 				field.Invalid(specField.Child("priorityClassName"), "invalid_class", ""),
 			},
 		},
+		"should have priority once priorityClassName is set": {
```

Comment:
add a test for nil priority when priorityClassName is not set

Reply:
Done